src/js/saspowerschoolff.js

```diff
@@ -162,6 +171,12 @@ function main_page () {
         },
     }).$mount(".hypo-grade-div-fixed");
 }
+function show_cumulative_gpa (courses) {
+    $("#calculateCumulative").hide();
+    calculate_cumulative_gpa(courses).then(cumulative_gpa => {
+        $("table[border='0'][cellpadding='3'][cellspacing='1'][width='100%']").prepend(`<tr><td align="center">Cumulative GPA: ${cumulative_gpa.toFixed(2)}</td></tr>`);
```

Comment:
```suggestion
        $("table[border='0'][cellpadding='3'][cellspacing='1'][width='100%']").prepend(`<tr><td align="center">Cumulative GPA (9-12): ${cumulative_gpa.toFixed(2)}</td></tr>`);
```
I think this makes it a little clearer. Just a suggestion though, if you think this is clear enough as is, just keep it as is.